src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Offender.kt

```diff
@@ -76,34 +76,13 @@ data class Offender(
   @JoinColumn(name = "ROOT_OFFENDER_ID", updatable = false, insertable = false)
   var rootOffender: Offender? = null,
 
-  @Deprecated("Do not use; only some of the bookings may be associated with the root or any particular alias")
-  @OneToMany(mappedBy = "offender", cascade = [CascadeType.ALL], fetch = FetchType.LAZY, orphanRemoval = true)
-  val bookings: MutableList<OffenderBooking> = mutableListOf(),
+  // CAUTION: this list is only populated for the root offender; normally the function getAllBookings() below should be used instead
+  @OneToMany(mappedBy = "rootOffender", cascade = [CascadeType.ALL], fetch = FetchType.LAZY, orphanRemoval = true)
+  private val allBookings: MutableList<OffenderBooking> = mutableListOf(),
```

Comment:
so was it not possible to map this in JPA land for all offender aliases ? (I can imagine some sort cascading delete error ??)